test/__snapshots__/TickLens.spec.ts.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`TickLens #getPopulatedTicksInWord fully populated ticks 1`] = `2803889`;
+exports[`TickLens #getPopulatedTicksInWord fully populated ticks 1`] = `2959038`;
 
-exports[`TickLens #getPopulatedTicksInWord gas for single populated tick 1`] = `55664`;
+exports[`TickLens #getPopulatedTicksInWord gas for single populated tick 1`] = `90708`;
```

Comment:
Kinda huge increase. TickLens isn't meant to be used on-chain so not a huge deal, but still worth investigating